sdk/Lusid.Sdk.Tests/tutorials/Instruments/EquityOptionExamples.cs

```diff
@@ -81,7 +81,7 @@ protected override void CreateAndUpsertMarketDataToLusid(string scope, ModelSele
             Assert.That(cashflows.Count, Is.EqualTo(1));
         }
 
-        [LusidFeature("F22-11")]
+        [LusidFeature("F22-19")]
```

Comment:
Looks like referring to FX Option in the [features.yml MR](https://gitlab.com/finbourne/salesengineering/lusid-features/-/merge_requests/33/diffs)
- L400 "FX Option and Valuation"